fix: LEAP-659: Fix webpack styles processing for CSS

nick-skriabin opened the pull request:
Fixes the way how we process external css files. Those are not supposed to be prefixed.

Also fixes: LEAP-660, LEAP-661, LEAP-662, LEAP-663, LEAP-664

### PR fulfills these requirements
- [X] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [X] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [X] Self-reviewed and ran all changes on a local instance (for bug fixes/features)



#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [X] Frontend



#### What is the new behavior?
External CSS files fill not be prefixed anymore


#### What is the current behavior?
External CSS files receive a prefix which makes them unusable



### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [X] No
- [ ] Not sure (briefly explain the situation below)



### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit



### Which logical domain(s) does this change affect?
Whole LSO

nick-skriabin committed:
Fix webpack styles processing for CSS

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7845026514

nick-skriabin committed:
Formatting

nick-skriabin committed:
Safety first

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7845378344

nick-skriabin committed:
Merge branch 'develop' into fb-leap-659/styles-crash

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7846110507

nick-skriabin committed:
Exclude css modules from the prefixing process

robot-ci-heartex commented:
Hi @nicholasrq!

Unfortunately you don't have membership in HumanSignal organization, your PR wasn't synced with HumanSignal/label-studio-enterprise.

nick-skriabin committed:
Add antd styles

nick-skriabin committed:
Merge branch 'develop' into fb-leap-659/styles-crash

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7872016548
